_sass/minimal-mistakes/_reset.scss

```diff
@@ -0,0 +1,187 @@
+/* ==========================================================================
+   STYLE RESETS
+   ========================================================================== */
+
+* { box-sizing: border-box; }
+
+html {
+  /* apply a natural box layout model to all elements */
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-size: 12px;
+
+  @include breakpoint($medium) {
+    font-size: 14px;
+  }
+
+  @include breakpoint($large) {
+    font-size: 16px;
+  }
+
+  @include breakpoint($x-large) {
+    font-size: 18px;
```

Comment:
lines 11, 14, 18, and 22 are what changed. I probably should have brought in the files and committed first and then changed the sizes for a clearer history. Previously:
16px
18px
20px
22px